de-lib/src/main/java/org/iplantc/de/theme/base/client/collaborators/GroupDisplayStrings.properties

```diff
@@ -1,3 +1,5 @@
+addGroup=Add List
+deleteGroup=Delete List
 deleteGroupConfirm=By deleting the collaborator list {0}, any data that has been shared with this list will immediately be un-shared.  Do you wish to continue deleting the collaborator list?
```

Comment:
I agree; we're going to have to get some input from the science team on this. The distinction between sharing with a group and sharing with an individual user is clear to me, but I really don't know if that distinction will be clear to DE users.